stock_quant_sale_order/wizard/stock_quant_sale_order.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class QuantSaleOrderWizard(models.TransientModel):
+    _name = 'quant.sale.order.wizard'
+
+    partner_id = fields.Many2one(
+        'res.partner',
+        string='Customer',
+        required=True,
+    )
+
+    @api.multi
+    def action_stock_quant_saleorder(self):
+        '''This methos create a sale order.'''
+        context = dict(self._context or {})
+        stock_quant_obj = self.env['stock.quant']
+        sale_order_obj = self.env['sale.order']
+        sale_order_line_obj = self.env['sale.order.line']
+
+        active_ids = context.get('active_ids', [])
+        quant_ids = stock_quant_obj.browse(active_ids)
+        source_location = quant_ids[0].location_id
+
+        if any(q.location_id != source_location for q in quant_ids):
+            raise UserError(_('Please select quants that are in the same '
+                              'Location.'))
+
+        if any(q.sale_order_id != False for q in quant_ids):
```

Comment:
This condition seems to be always "True", can you kindly check the behavior?

Reply:
Fixed.